jax/lax_linalg.py

```diff
@@ -271,3 +278,36 @@ def qr_jvp_rule(primals, tangents, full_matrices):
 qr_p.def_abstract_eval(qr_abstract_eval)
 xla.translations[qr_p] = qr_translation_rule
 ad.primitive_jvps[qr_p] = qr_jvp_rule
+
+
+# SVD decomposition
+
+def svd_impl(operand, full_matrices, compute_uv):
+  s, u, vt = xla.apply_primitive(svd_p, operand, full_matrices=full_matrices, compute_uv=compute_uv)
+  return core.pack((s, u, vt))
+
+def svd_translation_rule(c, operand):
```

Comment:
You'll need to teach the translation rule about `full_matrices` and `compute_uv` keyword arguments.